README.md

```diff
@@ -6,18 +6,7 @@ Table of Contents
 =================
 
 * [Using this repo](#using-this-repo)
-* [Building a collector](#building-a-collector)
-* [Building with Cloud Build and Artifact Registry](#building-with-cloud-build-and-artifact-registry)
-* [Running on GKE](#running-on-gke)
-   * [Prerequisites](#prerequisites)
-   * [Set up namespace](#set-up-namespace)
-   * [Create ConfigMap](#create-configmap)
-   * [Build a container image](#build-a-container-image)
-   * [(Optional) Cleanup](#optional-cleanup)
-      * [Remove Kubernetes resources](#remove-kubernetes-resources)
-   * [Troubleshooting](#troubleshooting)
-      * [Permission errors exporting telemetry](#permission-errors-exporting-telemetry)
-      * [ImagePullBackOff error when deploying collector](#imagepullbackoff-error-when-deploying-collector)
+* [Sample Recipies](#sample-recipies)
```

Comment:
`Recipes`

Reply:
it would look nice to have sub-bullets for each recipe up here in the ToC

I used a [toc generator](https://github.com/ekalinin/github-markdown-toc) for this. maybe we should add that to the repo/precommit

Reply:
TOC fixed.